### ISSUE TITLE
More config options, please

### ISSUE THREAD

**Issue by terrapin47** [MinecraftModDevelopmentMods/BaseMetals#360]
If possible, it would be great to see configuration options for:

Removing / Hiding in JEI/Furnace recipes the following for ores
Crystals
Casings
Dense Plates
Shards
Dirty Powder
Clumps
Crushed Purified
Crushed

Also, disabling individual material's tconstruct integration as well.

Thanks.

**Comment by dshadowwolf** [comment]
This is actually being worked on. We've been working on implementing a simplified variant of JSON for handling a black-/white- list situation for disabling things at a very fine-grained level. The parser and base code for checking for something presence on one of the lists is basically complete, but needs some more testing and proofing before we move forward with documenting the format and merging it in.

**Comment by terrapin47** [comment]
Nice.  I'm also noticing something I'll add here - disabling the crack hammer does not remove its recipes from JEI, requiring you to fill in all the recipes under DisabledCrackHammerRecipes.  

This is both with disable_crack_hammer=true and "Enable Crackhammer"=false set.

**Comment by jriwanek** [comment]
Yes, that's correct, it's because the crusher registry is used for more than just the crackhammer, you can limit it somewhat by turning off automatic recipes however.

**Comment by terrapin47** [comment]
Oh, what else is it used for?  If I don't want that functionality I want to make sure I'm hitting all the right places.

**Comment by dshadowwolf** [comment]
The *Advantage line of mods also utilize it. At the moment we haven't gotten them ported b/c of non-familiarity with the capabilities systems, but we are working on that.

**Comment by jriwanek** [comment]
Actually we have for all but one, but they're not released for 1.12.x yet.

**Comment by terrapin47** [comment]
Hmm, is a config option to disable the crusher registry a possibility?

**Comment by jriwanek** [comment]
Possibly, we shall see, I suspect it would cause no end of confusion though.

**Comment by dshadowwolf** [comment]
Completely disabling the crusher registry is not, really, possible as its used by several mods, including the *Advantage series (as previously noted). Beyond that, we have a good start on the basis of a more flexible and wide-spread blacklisting system which will be part of the next large release. (the 3.x series)
